src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -0,0 +1,473 @@
+        max-y (+ y height)]
+    (and
+     (and
+      (or ignore-min-x? (>= locationX x))
+      (or ignore-max-x? (<= locationX max-x)))
+     (and
+      (or ignore-min-y? (>= locationY y))
+      (or ignore-max-y? (<= locationY max-y))))))
+
+(def record-audio-worklets (js/require "../src/js/record_audio_worklets.js"))
+
+(defn- ring-scale [scale substract]
+  (.ringScale ^js record-audio-worklets
+              scale
+              substract))
+
+(defn- record-button-big []
+  [:f>
+   (fn []
+     (let [scale (reanimated/use-shared-value 1)
+           opacity (reanimated/use-shared-value 0)
+           opacity-from (if @ready-to-lock? opacity-from-lock opacity-from-default)
+           animations (map
+                       (fn [index]
+                         (let [ring-scale (ring-scale scale (* scale-padding index))]
+                           {:scale ring-scale
+                            :opacity (reanimated/interpolate ring-scale [1 scale-to-each] [opacity-from 0])}))
+                       (range 0 5))
+           rings-color (cond
+                         @ready-to-lock? (colors/theme-colors colors/neutral-80-opa-5-opaque colors/neutral-80)
+                         @ready-to-delete? colors/danger-50
+                         :else colors/primary-50)
+           translate-y (reanimated/use-shared-value 0)
+           translate-x (reanimated/use-shared-value 0)
+           button-color colors/primary-50
+           icon-color (if (and (not (colors/dark?)) @ready-to-lock?) colors/black colors/white)
+           icon-opacity (reanimated/use-shared-value 1)
+           red-overlay-opacity (reanimated/use-shared-value 0)
+           gray-overlay-opacity (reanimated/use-shared-value 0)
+           complete-animation #(cond
+                                 (and @ready-to-lock? (not @record-button-is-animating?))
+                                 (do
+                                   (reset! locked? true)
+                                   (reset! ready-to-lock? false))
+                                 (and (not @locked?) (not @reviewing-audio?))
+                                 (do
+                                   (reset! recording? false)
+                                   (reset! ready-to-send? false)
+                                   (reset! ready-to-delete? false)
+                                   (reset! ready-to-lock? false)
+                                   (utils/clear-interval @recording-timer)
+                                   (reset! recording-length-ms 0)))
+           start-animation (fn []
+                             (reanimated/set-shared-value opacity 1)
+                             (reanimated/animate-shared-value-with-timing scale 2.6 signal-anim-duration :linear)
+                             ;; TODO: Research if we can implement this with withSequence method from Reanimated 2
+                             ;; GitHub issue [#14561]: https://github.com/status-im/status-mobile/issues/14561
```

Comment:
something we should discuss further as a team but maybe we don't need TODO's at all and the issue is enough. Keeps the codebase cleaner. 
@ilmotta you mentioned this before

Reply:
One thing that I like about TODO's is that we can find and keep track of all the tech debt easily by searching TODO keywords in the project. But yeah we can definitely discuss it and reach consensus on how to handle them, I can adapt myself.

Reply:
Okay do as you feel best for now and let's have a vote on how we want to approach this going forward. :) cc @OmarBasem @smohamedjavid As I commented something similar to you in other prs

Reply:
I have mixed feelings, because what usually happens over time is that we tend to produce more TODO comments than actually fixing them. They are often written in a poor way too (not the case in this PR). Eventually, they become less and less useful because of the large amount of TODO comments.

Reply:
Agree as well 
<img width="428" src="https://user-images.githubusercontent.com/18485527/208151599-7dc8566c-d34c-450c-aa88-79be6f49f86d.png">